doc/api-extensions.md

```diff
@@ -1445,3 +1445,6 @@ Adds support for refresh during volume migration.
 
 ## network\_counters\_errors\_dropped
 This adds the received and sent errors as well as inbound and outbound dropped packets to the network counters.
+
+## metrics
+This adds metrics to LXD.
```

Comment:
Could you expand this with info on the URL endpoint that was added and explain that the metrics are usable with Prometheus?